backtrace-library/src/main/java/backtraceio/library/common/RequestHelper.java

```diff
@@ -0,0 +1,53 @@
+package backtraceio.library.common;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+import backtraceio.library.logger.BacktraceLogger;
+
+public class RequestHelper {
+    private static final transient String LOG_TAG = RequestHelper.class.getSimpleName();
+
+    private static final String CRLF = "\r\n";
+
+    /**
+     * Get Content-Type of request
+     *
+     * @return string with content type and information about boundary
+     */
+    public static String getContentType() {
+        return "application/json";
+    }
+
+    /**
+     * Write JSON string to output data steam
+     *
+     * @param outputStream output data stream
+     * @param json         JSON string with BacktraceData object
+     * @throws IOException
+     */
+    public static void addJson(OutputStream outputStream, String json) throws IOException {
+        if (json == null || json.isEmpty() || outputStream == null) {
+            BacktraceLogger.w(LOG_TAG, "JSON is null/empty or output stream is null");
```

Comment:
We're printing too much information about different parameters. When a customer will have a problem with nullable string and we print this message to stdout, how you will try to fix this problem? It might be related to empty json (which is a different kind of problem to solve that empty stdout)

Reply:
I don't know when we might have nullable stream, I just used @BartoszLitwiniuk code:
https://github.com/backtrace-labs/backtrace-android/blob/dcf0eb97675d693c5aff0f733d407197e9dad168/backtrace-library/src/main/java/backtraceio/library/common/MultiFormRequestHelper.java#L55
I can separate error messages in both of these places @konraddysput